source/_components/media_player.enigma2.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: page
+title: "Enigma2"
+description: "Instructions on how to integrate an Enigma2 based box running OpenWebif into Home Assistant."
+date: 2019-02-21 12:02
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: openwebif.png
+ha_category: Media Player
+featured: false
+ha_release: 0.90
+ha_iot_class: "Local Polling"
+---
+
+The `enigma2` platform allows you to control a Linux based set-top box which is running [Enigma2](https://github.com/oe-alliance/oe-alliance-enigma2) with the OpenWebif plugin installed.
+
+[OpenWebif](https://github.com/E2OpenPlugins/e2openplugin-OpenWebif) is an open source web interface for Enigma2 based set-top boxes.
+
+To add a set-top box to your installation, add the following to your `configuration.yaml` file:
```

Comment:
Worth noting about the autodiscovery here, too?

Reply:
Good point, yes I will add that.